File: 9-palindrome-number/palindrome-number.java
class Solution {
    public boolean isPalindrome(int x) {
        if(x<0) return false;
        int copy=x;
        int sum=0;
        while(x!=0){
            int rem=x%10;
            sum=sum*10+rem;
            x/=10;
        }
        if(sum==copy) return true;
        else return false;
    }
}

File: 9-palindrome-number/palindrome-number.py
class Solution:
    def isPalindrome(self, x: int) -> bool:
        if x < 0 :
            return False
        res=0
        n=x
        while n!=0:
            rem=n%10
            res=res*10+rem
            n//=10
        print(res)
        return res==x
        